docs/asciidoc/static/codec.asciidoc

```diff
@@ -0,0 +1,13 @@
+:register_method:	true
+:encode_method:		true
+:decode_method:		true
+:plugintype:    	codec
+:pluginclass:   	Codecs
+:pluginname:    	example
+:pluginnamecap: 	Example
+
+:getstarted: For this example, we are going to examine the example codec plugin at https://github.com/logstash-plugins/logstash-codec-example/ .
+
+:methodheader: Logstash codecs require the `register` method, and one or both of the `decode` and `encode` methods.
```

Comment:
I'd say "must implement" rather than "require". "One or both" seems awkward to me...maybe "and the decode method  or the encode method (or both). The comma before the "and" is not necessary.